src/app.js

```diff
@@ -0,0 +1,79 @@
+import axios from "axios";
+
+// [IMAGE: flag]
+// [country-name]
+// [country-naam] is situated in [subarea-name]. It has a population of [amount] people.
+// The capital is [city] and you can pay with [currency]'s
+
+//https://restcountries.com/v3.1/all
+
+async function fetchCountries(name) {
+
+
+    try {
+    const response = await axios.get(`https://restcountries.com/v3.1/name/${name}?fullText=true`);
+
+        console.log(response.data[0]);
+
+        showCountryData(response.data[0]);
+
+        // maak searchField leeg na elke zoek opdracht
+        document.getElementById("searchField").value = "";
+    } catch (error) {
+        console.error(error.response.status);
+
+        if (error.response.status === 404) {
+            const errorElement = document.getElementById("inject_country");
+            errorElement.innerHTML = `
+            
+            <p style="color:red">Dit land konden we niet vinden probeer opnieuw!</p>
+            `
+        }
+    }
+}
+
+function showCountryData(country) {
```

Comment:
```suggestion
function showCountryData({ name, flags, subregion, population, capital, currencies }) {
```
Dan hoef je niet overal `country.` vóór te zetten 😉